### ISSUE TITLE
[Vehicle] Player vehicle change their walk/run settings

### ISSUE THREAD

**Issue by Rushor** [dalaranwow/dalaran-wow#3904]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (OLD - ... tags should no longer be used)
[//]: # (Priority - ... tags are assigned by team members)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Description**:
Vehicles which are controlled by player are sometimes very slow. (They change the run-setting to walk-setting)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Vehicles which are controlled by player are sometimes very slow. (They change the run-setting to walk-setting)

**Expected behaviour**: Tell us what should happen instead
The vehicle should not change the walk/run-settings.

**Steps to reproduce the problem**:

1. Example for one quest: https://github.com/dalaranwow/dalaran-wow/issues/3901
After dragging mines for the quest Generosity Abounds, if the Lithe Stalker uses any abilities, he'll be forced to walk by default, disabling him from running. That ofc can be fixed by pressing the walk/run button.
2. Another quest example https://github.com/dalaranwow/dalaran-wow/issues/2412
While this doesn't make sense I figured out what he was trying to tell me. I hit the spacebar and it then allowed me to move fast on my drake, as well as up and down.

3. 

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
http://wotlk.openwow.com/quest=13146
http://www.wowhead.com/quest=12856/cold-hearted

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Comment by Rushor** [comment]
Seel also: https://github.com/dalaranwow/dalaran-wow/issues/2878 and https://github.com/dalaranwow/dalaran-wow/issues/5362

**The Drakkensryd**

[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
On this quest, I was hoping around from drake to drake and killing the npc's on the drake just fine. Then I completed the quest. After that I used the quest item to go to the column ornament so I could get my feet back on the ground. I was pulled over to it just fine but now I'm stuck and can't move. I tried grappling over to a different column ornament but it wouldn't let me. I typed /reload thinking that might help but I'm still stuck. I'm not able to even use the hearthstone. :(  So I'm sitting here not able to do a thing. Can someone please help me? 
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Jump to drakes, fight red npcs.
2. Then I jumped on the closest drake to the pillar. it was pretty much even with the pillars floor.
3. Grapple over to the Column Ornament(A glowing crystal ball that is attached to one of the smaller pillars.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
